hibernate-core/src/main/java/org/hibernate/engine/jdbc/internal/BasicFormatterImpl.java

```diff
@@ -507,9 +494,22 @@ private static boolean isWhitespace(String token) {
 
 		private void newline() {
 			result.append( System.lineSeparator() )
-					.append( INDENT_STRING.repeat(indent) );
+					.append( INDENT_STRING.repeat( indent ) );
 			beginLine = true;
 		}
+
+		private void appendUntilToken(String stopToken) {
+			StringBuilder sb = new StringBuilder();
+			String t;
+			sb.append( this.token );
```

Comment:
Would this be better?

```suggestion
			final StringBuilder sb = new StringBuilder( this.token );
			String t;
```

Reply:
That's right.